titans/model/moe/widenet.py

```diff
@@ -36,33 +36,33 @@ def __init__(self,
         embedding = VanillaPatchEmbedding(img_size=img_size,
                                           patch_size=patch_size,
                                           in_chans=in_chans,
-                                          embed_size=d_model)
+                                          embed_size=hidden_size)
         embed_dropout = Dropout(p=drop_rate, mode=ParallelMode.TENSOR)
 
         shared_sa = SelfAttentionForMoe(**moe_sa_args(
-            d_model=d_model, n_heads=num_heads, d_kv=d_kv, attention_drop=attention_drop, drop_rate=drop_rate))
+            hidden_size=hidden_size, n_heads=num_heads, d_kv=d_kv, attention_drop=attention_drop, drop_rate=drop_rate))
 
         noisy_func = NormalNoiseGenerator(num_experts)
         shared_router = Top2Router(capacity_factor_train=capacity_factor_train,
                                    capacity_factor_eval=capacity_factor_eval,
                                    noisy_func=noisy_func,
                                    drop_tks=drop_tks)
-        shared_experts = build_ffn_experts(num_experts, d_model, d_ff, drop_rate=drop_rate)
+        shared_experts = build_ffn_experts(num_experts, hidden_size, d_ff, drop_rate=drop_rate)
 
         # stochastic depth decay rule
         dpr = [x.item() for x in torch.linspace(0, drop_path, depth)]
         blocks = [
             TransformerLayer(att=shared_sa,
-                             ffn=MoeLayer(dim_model=d_model,
+                             ffn=MoeLayer(dim_model=hidden_size,
```

Comment:
For the consistency, I think dim_model should be hidden_size here.

Reply:
This is because the class MoeLayer uses dim_model in the module colossalai.